nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -418,6 +420,12 @@ sub filterUnits {
 print STDERR "setting up tmpfiles\n";
 system("@systemd@/bin/systemd-tmpfiles", "--create", "--remove", "--exclude-prefix=/dev") == 0 or $res = 3;
 
+# Services depending on DynamicUser needs nscd to resolve users/groups
```

Comment:
Not sure I fully understand what this does, but I guess it ensures nscd is restarted before other services (is that right?) 

If so, we should extend this comment a bit further to describe this.